docs/concepts/networks.mdx

```diff
@@ -1,42 +1,39 @@
 ---
 title: Networks
-description: 'Combine one or more agents into a Network'
+description: 'Combine one or more agents into a Network.'
 icon: 'chart-network'
 iconType: 'regular'
 ---
 
-Networks are **Systems of [Agents](/concepts/agents)**. Use Networks to create complex, stateful workflows with one or more Agents.
+Networks are **Systems of [Agents](/concepts/agents)**. Use Networks to create powerful AI Agents by combining multiple Agents.
```

Comment:
create AI Agents by combining agents? I feel like we're describing the thing with the same thing. Suggesting to find another word